.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: Lint and Test
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Test
+    timeout-minutes: 2
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          ref: ${{ github.ref }}
+          persist-credentials: false
```

Comment:
What's the `ref` and `persist-credentials` for, do you have private dependencies? I would have just expected:

```yml
- name: Checkout
   uses: Brightspace/third-party-actions@actions/checkout
```

Reply:
We did this in another action on my team, so I was under the impression it was required. If it's not, I'll remove it.